### DISCUSSION TITLE
3D Editor: Focus Raycast Position

### DISCUSSION THREAD

**Discussion by Mirex97** [godotengine/godot-proposals#8985] [Editor]
Currently there are options to focus on origin and object position, but what I would want is to focus on a raycasted position instead. This way you'd have more control over the orbit position of the camera in the scene instead of having to try and align the position with middle click and dragging (including alt + middle click dragging).

This would be similar to how in Blender you can set the orbit position with Alt + Middle click. And the camera would then span over to the clicked position.

I tried to make a plugin out of this but wasn't able to modify the position that the camera orbits around. And modifying the editor's Camera3D position does not store the position if you then use middle click to rotate the camera view (returns back to the orbit position).

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/910.
